static/src/javascripts/projects/common/modules/commercial/commercial-features.ts

```diff
@@ -42,7 +68,10 @@ class CommercialFeatures {
 		const isIdentityPage =
 			config.get('page.contentType') === 'Identity' ||
 			config.get('page.section') === 'identity'; // needed for pages under profile.* subdomain
-		const switches = config.get<Record<string, boolean>>('switches', {});
+		const switches = config.get<Record<string, boolean | undefined>>(
```

Comment:
I don't think this type change is necessary or accurate: keys with `undefined` values will not be present.

Reply:
Under test conditions `switches.commercial` was `undefined`, I did initially just coerce to a boolean, but the type is actually wrong